Resources/Prototypes/_Goobstation/MartialArts/cqc.yml

```
@@ -0,0 +1,69 @@
+  - Harm
+  event: !type:CqcSlamPerformedEvent { }
+  extraDamage: 10 #
+  paralyzeTime: 12 # in seconds
+
+
+- type: combo
+  id: CQCKick
+  name: Kick
+  martialArtsForm: CloseQuartersCombat
+  attacks:
+  - Harm
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)